products/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+class Product(models.Model):
+    name           = models.CharField(max_length=30)
+    country        = models.CharField(max_length=30)
+    price          = models.IntegerField()
+    receiving_date = models.DateTimeField(auto_now_add=True)
+    description    = models.CharField(max_length=200)
+    thumb_image    = models.CharField(max_length=200)
```

Comment:
```suggestion
    thumbnail_image_url    = models.CharField(max_length=200)
```
데이터베이스 테이블명 및 컬럼명은 혼선은 방지하기 위해 줄이지 않고 있는 그대로 지정해주시는 걸 추천드립니다. 워낙 예민한 정보이고 추후 여러개의 서비스와 연결되어 사용할수도 있게 되는 부분이라서요.